CLI tools/CurseForge to Packwiz-Modrinth.py

```diff
@@ -30,12 +30,12 @@ def extract_file(from_zip, from_file, to_path, from_desc, to_desc):
             print("Skipped " + from_desc + " copying to " + to_desc + ", didn't exist")
 
 mod_files = os.listdir(mods_path)
-if refresh_only == False:
+if not refresh_only:

```

Comment:
Interesting, didn't know that syntax in Python. Is there a performance difference as well?

Reply:
Nope, there is no performance difference, it's just (!=) operator, works same as in any other languages

Reply:
Would `if !refresh_only` work as well? I'm not too familiar with Python syntax

Reply:
No, that's why `not` is used here. It's python variant of (!)